src/js/components/agencyV2/statusOfFunds/VisualizationSection.jsx

```diff
@@ -6,18 +6,38 @@
 // eslint-disable-next-line no-unused-vars
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Pagination } from 'data-transparency-ui';
 import { levels } from './StatusOfFunds';
+import StatusOfFundsChart from '../visualizations/StatusOfFundsChart';
 
 const propTypes = {
     level: PropTypes.number,
     agencyId: PropTypes.string,
-    fy: PropTypes.string
+    agencyName: PropTypes.string,
+    fy: PropTypes.string,
+    data: PropTypes.object
 };
 
-const VisualizationSection = ({ level }) => (
-    <>
-        DEV-8049 horizontal bar chart viewing {levels[level]}s
-    </>
+const VisualizationSection = ({
+    level,
+    agencyName,
+    fy,
+    data
+}) => (
+    <div className="status-of-funds__visualization">
+        <h6>{agencyName} by <strong>{levels[level]}</strong> for FY {fy}</h6>
```

Comment:
Eventually `agencyName` should be updated to display either the agency name or the Sub-Component name (from Redux) depending on the level. We can do this as part of the click-through functionality ticket (DEV-8052)!